sdk/storage/ci.yml

```diff
@@ -37,7 +37,7 @@ stages:
       ServiceDirectory: storage
       # TODO: Change to azure-storage once we have an strategy to run livetests or use test recordings
       CtestRegex: azure-core
-      Coverage: disabled
+      Coverage: enabled
```

Comment:
This needs to stay disabled. It is for CI tests only.
CodeCoverage will be ON for LiveTests by default.  I have enabled it here: https://github.com/Azure/azure-sdk-for-cpp/pull/939

Reply:
Closing the PR.